### PR TITLE
Fixing security concerns with one of the links

### PR DESCRIPTION
**Pull request by 9ebd7134**
**Fixes**: #

🚨 Please review the [guidelines for contributing](CONTRIBUTING.md) and our [code of conduct](../CODE_OF_CONDUCT.md) to this repository. 🚨
**Please complete these steps and check these boxes (by putting an x inside the brackets) before filing your PR:**

- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

#### Short description of what this resolves:

TLDR; 
Fixed non-secure link

#### Proposed changes:

- Changed the link

👍 Thank you!

**Quick Note**
Please don't allow this since fixing one little link shoudn't be enough. Hope it helps people in the future :) im going to fix more issues soon

### CODE REVIEW

**Comment by thedaviddias** [comment]
Nice try @9ebd7134

**Comment by 9ebd7134** [comment]
> Nice try @9ebd7134

hmm ? have you tried visiting https://varvy.com/robottxt.html
```
Warning: Potential Security Risk Ahead

Firefox detected an issue and did not continue to varvy.com. The website is either misconfigured or your computer clock is set to the wrong time.

It’s likely the website’s certificate is expired, which prevents Firefox from connecting securely. If you visit this site, attackers could try to steal information like your passwords, emails, or credit card details.

What can you do about it?

The issue is most likely with the website, and there is nothing you can do to resolve it. You can notify the website’s administrator about the problem.

Learn more…
```
